app/src/main/java/it/ministerodellasalute/immuni/ui/dialog/BottomSheetDialogLightFragment.kt

```diff
@@ -51,7 +51,7 @@ abstract class BottomSheetDialogLightFragment : BottomSheetDialogFragment() {
         val window: Window? = dialog?.window
         if (window != null) {
             val metrics = DisplayMetrics()
-            window.windowManager.defaultDisplay.getMetrics(metrics)
+            window.windowManager.defaultDisplay.getRealMetrics(metrics)
```

Comment:
@marcouberti should be good now, it has the intended behaviour, is it possible to check it?

Reply:
Nope, I still have the following behaviour with your change: dark nav bar. Weird that only on your device there is this issue... 
![device-2020-05-27-002517](https://user-images.githubusercontent.com/3175499/82956196-9882f080-9fb0-11ea-8f6f-7e5d23c491e2.png)